### DISCUSSION TITLE
Move Preferences from the Edit menu to the File menu

### DISCUSSION THREAD

**Discussion by GsMumbo** [maforget/ComicRackCE#28] [Ideas]
In most software the Edit menu is typically used for manipulating individual items. File is typically used for manipulating entire files, but also contains some of the higher level actions. 

Would it be possible (and should we) move Preferences under the Files menu? It would take reworking muscle memory for a lot of CR users but it would make it a lot more standardized and accessible for new users.

**Comment by maforget** [comment]
> In most software the Edit menu is typically used for manipulating individual items.

Actually in a lot of software, Preferences are under Edit. Photoshop, Premiere, Blender to name a few.

**Comment by Totengeist** [comment]
In my experience, the Preferences item is most commonly found under Tools or Edit. I've rarely seen it under the File menu, except in recent (2016+, I think) versions of Microsoft Office and applications that don't have use for Edit or Tools menus.
